deprecation.py

```diff
@@ -121,10 +125,12 @@ def deprecated(deprecated_in=None, removed_in=None, current_version=None,
                           means immediate deprecation. If this is not
                           specified, then the `removed_in` and
                           `current_version` arguments are ignored.
-    :param removed_in: The version when the decorated method will be removed.
-                       The default is **None**, specifying that the function
-                       is not currently planned to be removed.
-                       Note: This cannot be set to a value if
+    TODO: Add Documentation on date input for removed_in DONE
+    :param removed_in: The version or date when the decorated method will be
+                       removed. The default is **None**, specifying that
+                       the function is not currently planned to be removed.
+                       The date input is of type datetime.date
```

Comment:
You could say "or datetime.date" above and save the extra sentence.

Reply:
Changed